app/src/main/java/org/fossasia/pslab/communication/peripherals/MCP4728.java

```diff
@@ -34,6 +34,7 @@
     private List<Integer> VREFS;
     public Map<String, DACChannel> CHANS = new LinkedHashMap<>();
     private Map<Integer, String> CHANNELMAP = new LinkedHashMap<>();
+    private Map<String, Double> values = new LinkedHashMap<>();
```

Comment:
`values` will look nice in uppercase as the above fields! What do you think? :blush: